cms/djangoapps/contentstore/views/course.py

```diff
@@ -469,6 +474,52 @@ def _get_rerun_link_for_item(course_key):
     return reverse_course_url('course_rerun_handler', course_key)
 
 
+def _deprecation_info(course_module, deprecated_block_types):
```

Comment:
@ormsbee I have also made this function generic so that in future we can pass it other deprecated block types.